src/mocompiler.js

```diff
@@ -148,20 +165,19 @@ Compiler.prototype._generateList = function () {
 /**
  * Calculate buffer size for the final binary object
  *
- * @param {import('./index.d.ts').GetTextTranslations} list An array of translation strings from _generateList
- * @return {Object} Size data of {msgid, msgstr, total}
+ * @param {import('./types.js').GetTextTranslation[]} list An array of translation strings from _generateList
+ * @return {{msgid: number, msgstr: number, total: number}} Size data of {msgid, msgstr, total}
```

Comment:
I would create a type of this data structure.

Reply:
sure good catch!